agot-bg-game-server/src/common/ingame-game-state/game-data-structure/GameLog.ts

```diff
@@ -224,6 +225,26 @@ interface ArmiesReconciled {
     armies: [string, string[]][];
 }
 
+interface HouseCardAbilityNotUsed {
+    type: "house-card-ability-not-used";
+    house: string;
+    houseCard: string;
+}
+
+interface PatchfaceUsed {
```

Comment:
The convention is for the type to be the name of the interface but in snake-case, so it would be `patchface-used` and  `doran-used`.

Reply:
All other interfaces are in PascalCase also and the type is kebab-case...

Reply:
Ah sorry.. I see the point. type is just "doran". I fix it.